lua/starfall/libs_sh/hologram.lua

```diff
@@ -293,14 +293,33 @@ else
 		end
 	end
 
+	local render_GetColorModulation, render_GetBlend = render.GetColorModulation, render.GetBlend
+	local render_SetColorModulation, render_SetBlend = render.SetColorModulation, render.SetBlend
+
 	--- Manually draws a hologram, requires a 3d render context
 	-- @client
-	function hologram_methods:draw()
+	-- @param bool? noTint If true, renders the hologram with it's color and opacity. This is the default behavior.
```

Comment:
`noTint If false or nil, renders the hologram with it's color and opacity at the cost of performance. This is the default behavior.`

Reply:
Regardless, `If true` is wrong

Reply:
Should have fixed that, let me know if there are any more issues.